From f134b06b6950e60c2e6052cc0a9511fa8857159d Mon Sep 17 00:00:00 2001
From: ariel aviad <aaviad@habana.ai>
Date: Sun, 22 Dec 2024 17:25:34 +0200
Subject: [PATCH] [FSW-6867] palad: remove pldm defconfig files

Remove pldm defconfig files
we no more support pldm platform we renamed it to palad

Change-Id: I23f0415376375bf38f155a86a39c812642908808
Signed-off-by: ariel aviad <aaviad@habana.ai>
---

diff --git a/configs/fs1_arm_pldm_defconfig b/configs/fs1_arm_pldm_defconfig
deleted file mode 100644
index c9092c2..0000000
--- a/configs/fs1_arm_pldm_defconfig
+++ /dev/null
@@ -1,19 +0,0 @@
-BR2_aarch64=y
-BR2_SVN="svn"
-BR2_DL_DIR="$(BR2_EXTERNAL_HABANALABS_PATH)/dl"
-BR2_PACKAGE_OVERRIDE_FILE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/local.mk"
-# BR2_TOOLCHAIN is not set
-BR2_INIT_NONE=y
-BR2_SYSTEM_BIN_SH_NONE=y
-# BR2_TARGET_GENERIC_GETTY is not set
-# BR2_PACKAGE_BUSYBOX is not set
-# BR2_PACKAGE_IFUPDOWN_SCRIPTS is not set
-# BR2_TARGET_ROOTFS_TAR is not set
-BR2_PIC_PIE=n
-BR2_SSP_NONE=y
-BR2_RELRO_NONE=y
-BR2_FORTIFY_SOURCE_NONE=y
-BR2_TOOLCHAIN_USES_GLIBC=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SUPPORT=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SIGNATURE_SUPPORT=y
\ No newline at end of file
diff --git a/configs/gaudi2_arc_pldm_defconfig b/configs/gaudi2_arc_pldm_defconfig
deleted file mode 100644
index fa2e65c..0000000
--- a/configs/gaudi2_arc_pldm_defconfig
+++ /dev/null
@@ -1,30 +0,0 @@
-BR2_USE_MMU=y
-BR2_arcle=y
-BR2_archs4x=y
-BR2_SVN="svn"
-BR2_DL_DIR="$(BR2_EXTERNAL_HABANALABS_PATH)/dl"
-BR2_SHARED_STATIC_LIBS=y
-BR2_PACKAGE_OVERRIDE_FILE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/local.mk"
-BR2_TOOLCHAIN_EXTERNAL=y
-BR2_TARGET_GENERIC_HOSTNAME=""
-BR2_TARGET_GENERIC_ISSUE=""
-BR2_INIT_NONE=y
-BR2_SYSTEM_BIN_SH_NONE=y
-# BR2_TARGET_GENERIC_GETTY is not set
-BR2_ROOTFS_POST_IMAGE_SCRIPT="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/$(HL_DEV)/post-image-spi.sh $(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/post-image-spi-arc_global.sh"
-# BR2_PACKAGE_BUSYBOX is not set
-# BR2_PACKAGE_IFUPDOWN_SCRIPTS is not set
-# BR2_TARGET_ROOTFS_TAR is not set
-BR2_PACKAGE_BAREMETAL=y
-BR2_PACKAGE_BAREMETAL_ARC_EXT_ROM=y
-BR2_PACKAGE_ARCBOOT=y
-BR2_PACKAGE_PID=y
-BR2_PACKAGE_MGMT=y
-BR2_PACKAGE_PREBOOT=y
-BR2_PIC_PIE=n
-BR2_SSP_NONE=y
-BR2_RELRO_NONE=y
-BR2_FORTIFY_SOURCE_NONE=y
-BR2_PACKAGE_HOST_DTC=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SUPPORT=y
diff --git a/configs/gaudi2_arm_pldm_defconfig b/configs/gaudi2_arm_pldm_defconfig
deleted file mode 100644
index 102a487..0000000
--- a/configs/gaudi2_arm_pldm_defconfig
+++ /dev/null
@@ -1,75 +0,0 @@
-BR2_aarch64=y
-BR2_SVN="svn"
-BR2_DL_DIR="$(BR2_EXTERNAL_HABANALABS_PATH)/dl"
-BR2_SHARED_STATIC_LIBS=y
-BR2_PACKAGE_OVERRIDE_FILE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/local.mk"
-BR2_GLOBAL_PATCH_DIR="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/patches"
-BR2_TOOLCHAIN_EXTERNAL=y
-BR2_TARGET_GENERIC_HOSTNAME="$(HL_DEV)"
-BR2_TARGET_GENERIC_ISSUE="Welcome to Gaudi2"
-BR2_TARGET_GENERIC_ROOT_PASSWD="root"
-BR2_SYSTEM_BIN_SH_BASH=y
-BR2_SYSTEM_DHCP="eth0"
-BR2_ROOTFS_OVERLAY="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/rootfs_overlay"
-BR2_ROOTFS_POST_BUILD_SCRIPT="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/post-build.sh"
-BR2_POST_IMAGE_FW_LOADER_ITS_TEMPLATE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/agent-fw_loader.its"
-BR2_POST_IMAGE_AGENTS_ITS_TEMPLATE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/$(HL_DEV)/agent.its"
-BR2_ROOTFS_POST_IMAGE_SCRIPT="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/post-image-fit.sh $(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/post-image-agents.sh"
-BR2_LINUX_KERNEL=y
-BR2_LINUX_KERNEL_USE_ARCH_DEFAULT_CONFIG=y
-BR2_LINUX_KERNEL_CONFIG_FRAGMENT_FILES="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/$(HL_DEV)/linux_$(HL_TARGET).fragment"
-BR2_LINUX_KERNEL_IMAGE_TARGET_CUSTOM=y
-BR2_LINUX_KERNEL_IMAGE_TARGET_NAME="Image.gz"
-BR2_LINUX_KERNEL_DTS_SUPPORT=y
-BR2_LINUX_KERNEL_INTREE_DTS_NAME="habanalabs/$(HL_DEV)/$(HL_TARGET)"
-BR2_PACKAGE_BUSYBOX_CONFIG="package/busybox/busybox-minimal.config"
-BR2_PACKAGE_BUSYBOX_CONFIG_FRAGMENT_FILES="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/busybox.fragment"
-BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
-BR2_PACKAGE_BUSYBOX_WATCHDOG=y
-BR2_PACKAGE_MTD=y
-# BR2_PACKAGE_MTD_MTD_DEBUG is not set
-# BR2_PACKAGE_MTD_NANDDUMP is not set
-# BR2_PACKAGE_MTD_NANDTEST is not set
-# BR2_PACKAGE_MTD_NANDWRITE is not set
-# BR2_PACKAGE_MTD_UBIATTACH is not set
-# BR2_PACKAGE_MTD_UBICRC32 is not set
-# BR2_PACKAGE_MTD_UBIDETACH is not set
-# BR2_PACKAGE_MTD_UBIFORMAT is not set
-# BR2_PACKAGE_MTD_UBIMKVOL is not set
-# BR2_PACKAGE_MTD_UBINFO is not set
-# BR2_PACKAGE_MTD_UBINIZE is not set
-# BR2_PACKAGE_MTD_UBIRENAME is not set
-# BR2_PACKAGE_MTD_UBIRMVOL is not set
-# BR2_PACKAGE_MTD_UBIRSVOL is not set
-# BR2_PACKAGE_MTD_UBIUPDATEVOL is not set
-# BR2_PACKAGE_MTD_UBIBLOCK is not set
-BR2_PACKAGE_I2C_TOOLS=y
-BR2_PACKAGE_LM_SENSORS=y
-BR2_PACKAGE_UBOOT_TOOLS=y
-BR2_PACKAGE_ZLIB=y
-# BR2_PACKAGE_IFUPDOWN_SCRIPTS is not set
-BR2_PACKAGE_KMOD=y
-BR2_TARGET_ROOTFS_CPIO=y
-BR2_TARGET_ROOTFS_CPIO_GZIP=y
-# BR2_TARGET_ROOTFS_TAR is not set
-BR2_TARGET_UBOOT=y
-BR2_TARGET_UBOOT_BOARD_DEFCONFIG="$(HL_DEV)"
-BR2_TARGET_UBOOT_CONFIG_FRAGMENT_FILES="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/$(HL_DEV)/uboot_$(HL_TARGET).fragment"
-BR2_TARGET_UBOOT_NEEDS_DTC=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SUPPORT=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SIGNATURE_SUPPORT=y
-BR2_PACKAGE_ARMCP=y
-# BR2_PACKAGE_BAREMETAL is not set
-BR2_PACKAGE_BTL=y
-BR2_PACKAGE_EEPROM_UTIL=y
-BR2_POST_IMAGE_UBOOT_ITS_TEMPLATE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/uboot_extended.its"
-BR2_PACKAGE_AGENT_PCIE=y
-BR2_PACKAGE_AGENT_SERDES=y
-BR2_PACKAGE_AGENT_HBM=y
-BR2_PACKAGE_AGENT_FWL=y
-BR2_PIC_PIE=n
-BR2_SSP_NONE=y
-BR2_RELRO_NONE=y
-BR2_FORTIFY_SOURCE_NONE=y
-BR2_PACKAGE_HOST_HL_TEST=y
diff --git a/configs/gaudi3_arc_pldm_defconfig b/configs/gaudi3_arc_pldm_defconfig
deleted file mode 100644
index b738de4..0000000
--- a/configs/gaudi3_arc_pldm_defconfig
+++ /dev/null
@@ -1,40 +0,0 @@
-BR2_USE_MMU=y
-BR2_arcle=y
-BR2_archs4x=y
-BR2_SVN="svn"
-BR2_DL_DIR="$(BR2_EXTERNAL_HABANALABS_PATH)/dl"
-BR2_SHARED_STATIC_LIBS=y
-BR2_PACKAGE_OVERRIDE_FILE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/local.mk"
-BR2_GLOBAL_PATCH_DIR="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/patches"
-BR2_TOOLCHAIN_EXTERNAL=y
-BR2_TARGET_GENERIC_HOSTNAME=""
-BR2_TARGET_GENERIC_ISSUE=""
-BR2_INIT_NONE=y
-BR2_SYSTEM_BIN_SH_NONE=y
-# BR2_TARGET_GENERIC_GETTY is not set
-BR2_ROOTFS_POST_IMAGE_SCRIPT="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/post-image-fit-arc.sh $(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/post-image-spi-arc_global.sh $(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/post-image-agents.sh $(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/$(HL_DEV)/create-fake-fuse-image.sh"
-# BR2_PACKAGE_BUSYBOX is not set
-# BR2_PACKAGE_IFUPDOWN_SCRIPTS is not set
-# BR2_TARGET_ROOTFS_TAR is not set
-BR2_PACKAGE_HOST_UBOOT_TOOLS=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SUPPORT=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SIGNATURE_SUPPORT=y
-BR2_PACKAGE_BAREMETAL=y
-BR2_PACKAGE_PCI_PIF_ARC=y
-BR2_PACKAGE_ARCBOOT=y
-BR2_PACKAGE_MGMT=y
-BR2_PACKAGE_PID=y
-BR2_PACKAGE_EVENT=y
-BR2_PACKAGE_PREBOOT=y
-BR2_PACKAGE_BAREMETAL_ARC_EXT_ROM=y
-BR2_PACKAGE_AGENT_FWL=y
-BR2_POST_IMAGE_FW_LOADER_ITS_TEMPLATE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/agent-fw_loader.its"
-BR2_POST_IMAGE_AGENTS_ITS_TEMPLATE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/$(HL_DEV)/agent.its"
-BR2_PIC_PIE=n
-BR2_SSP_NONE=y
-BR2_RELRO_NONE=y
-BR2_FORTIFY_SOURCE_NONE=y
-BR2_PACKAGE_PALAD_BLOADER=y
-BR2_PACKAGE_AGENT_HBM=n
-BR2_PACKAGE_AGENT_PCIE=n
-BR2_PACKAGE_AGENT_SERDES=n
diff --git a/configs/gaudi3_arm_pldm_defconfig b/configs/gaudi3_arm_pldm_defconfig
deleted file mode 100644
index 896bef2..0000000
--- a/configs/gaudi3_arm_pldm_defconfig
+++ /dev/null
@@ -1,21 +0,0 @@
-BR2_aarch64=y
-BR2_SVN="svn"
-BR2_DL_DIR="$(BR2_EXTERNAL_HABANALABS_PATH)/dl"
-BR2_PACKAGE_OVERRIDE_FILE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/common/local.mk"
-# BR2_TOOLCHAIN is not set
-BR2_INIT_NONE=y
-BR2_SYSTEM_BIN_SH_NONE=y
-BR2_ROOTFS_POST_IMAGE_SCRIPT="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/${HL_DEV}/post-image-spi.sh"
-# BR2_TARGET_GENERIC_GETTY is not set
-# BR2_PACKAGE_BUSYBOX is not set
-# BR2_PACKAGE_IFUPDOWN_SCRIPTS is not set
-# BR2_TARGET_ROOTFS_TAR is not set
-BR2_PIC_PIE=n
-BR2_SSP_NONE=y
-BR2_RELRO_NONE=y
-BR2_FORTIFY_SOURCE_NONE=y
-BR2_TOOLCHAIN_USES_GLIBC=y
-BR2_PACKAGE_HOST_HL_TEST=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SUPPORT=y
-BR2_PACKAGE_HOST_UBOOT_TOOLS_FIT_SIGNATURE_SUPPORT=y
